molpipeline/mol2any/mol2bool.py

```diff
@@ -50,3 +50,23 @@ def pretransform_single(self, value: Any) -> bool:
         if isinstance(value, InvalidInstance):
             return False
         return True
+
+    def transform_single(self, value: Any) -> Any:
+        """Transform a single molecule to the new representation.
+
+        RemovedMolecule objects are passed without change, as no transformations are applicable.
+
+        Parameters
+        ----------
+        value: Any
+            Current representation of the molecule. (Eg. SMILES, RDKit Mol, ...)
+
+        Returns
+        -------
+        Any
+            New representation of the molecule. (Eg. SMILES, RDKit Mol, Descriptor-Vector, ...)
```

Comment:
This doc-string also is too general. What will be returned is either the input molecule or an InvalidInstance. This should be mentioned here and not SMILES or descriptor vectors.

Reply:
The second part of your comment is actually not true. What will be returned is the transformed (and finalized) value or the transformed value if it is an InvalidInstance (which will likely never be reached. I will actually also remove this line and state that the transformed output value will be returned.

Reply:
Changed a few descriptions here. Please have a look